src/main/java/io/pravega/connectors/flink/PravegaInputSplit.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.pravega.connectors.flink;
+
+import com.google.common.base.Preconditions;
+import io.pravega.client.segment.impl.Segment;
+import org.apache.flink.core.io.InputSplit;
+
+/**
+ * A {@link PravegaInputSplit} corresponds to a Pravega {@link Segment}.
+ */
+public class PravegaInputSplit implements InputSplit {
```

Comment:
Is there any requirement that the split implement equality or comparison checks?

Reply:
I don't think so. Though, it would not hurt to implement that (pretty sure we'll need it eventually for the unit tests anyway). Will add.